src/VestingWalletWithCliffAndClawback.sol

```diff
@@ -53,166 +27,42 @@ contract VestingWalletWithCliffAndClawback is VestingWallet, Ownable2Step {
         uint64 durationSeconds,
         uint64 cliffDurationSeconds
     ) payable 
-        Ownable2Step()
         VestingWallet(beneficiaryAddress, startTimestamp, durationSeconds)
+        VestingWalletWithCliff(cliffDurationSeconds)
+        VestingWalletWithClawback(ownerAddress)
     {
```

Comment:
There should also be a linter hint in here, but the linter didn't pick up on it. Does it need to be reconfigured?

Insert:

```solidity
// solhint-disable-previous-line no-empty-blocks
```

Reply:
I just did the standard initialization, so maybe I didn't fully set up everything that _should_ be in it

Reply:
done